### PR TITLE
Added some nice lists!

### PR DESCRIPTION
**Pull request by 1w3j**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -290,6 +292,7 @@ List of useful, silly and [awesome](#awesome-) lists curated on GitHub. Contribu
 * [maintenance-modules](https://github.com/maxogden/maintenance-modules) – NPM / Node.js modules useful for maintaining or developing modules
 * [manong](https://github.com/nemoTyrant/manong) _In Chinese_ – Weekly digest of technology
 * [markdown-resources](https://github.com/rhythmus/markdown-resources) – Markdown resources: apps, dialects, parsers, people, …
+* [Marketing-for-Engineers](https://github.com/LisaDziuba/Marketing-for-Engineers) – Collection of marketing articles & tools to grow your product.
```
[the remark this answers is not in the document]
Comment (1w3j) [reply README.md]: of course

**Comment by 1w3j** [comment]
I resolved them.

**Comment by jnv** [comment]
@1w3j Thanks but I don't see a new commit or any changes... I will do it myself then.

**Comment by jnv** [comment]
Unfortunately I couldn't amend changes to your pull request directly, possibly because your fork has been deleted. I have committed your changes with my suggestions in 21b66b57ffe9b50a7ae8aebaebeb31d8a1a1dba0. Thanks again for your contribution!

**Pull request closed without merging**